guidelines/data.md

```diff
@@ -0,0 +1,20 @@
+**_Journal of Cheminformatics_ will only publish research or software that is entirely reproducible by third parties.
+This means that any datasets, software and algorithms that are required to reach the conclusions stated in the
+paper must be provided as supplemental materials, or be otherwise accessible without the need for registration,
+login or agreement with license terms other than Creative Commons licenses for data and text and
+[OSI-approved Open Source Licenses](http://opensource.org/licenses/alphabetical)
+for software. For any software, the source code must be provided.**
+
+Data articles should describe a curated, CCZero-license dataset likely to be of broad utility. The dataset must
```

Comment:
Line 4 mentions Creative Commons, here you specify CCZero, which is more restrictive than the general guidelines. Do we want to be so restrictive? This would eliminate e.g. CC-BY etc. One possibility could be to adjust to a CC dataset with a strong recommendation for ... ?

Reply:
Agreed - CCZero seems overly restrictive

Reply:
Updated.